CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 16.0.0 - ..2023
+## Features
+- ISY-144 Upgrade Angular und PrimeNG Bibliothek auf Version 16
```

Comment:
- Don't input a date when no release is done.
- Mark breaking changes as such, don't introduce them as a feature.